author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
+  attr_reader :first_name, :last_name, :id, :items
+
+  def initialize(first_name, last_name, id = nil)
+    @first_name = first_name
+    @last_name = last_name
+    @id = id || rand(1..1000)
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.genre = self
```

Comment:
As you are creating the author class, try by modifying this line into `item.add_author(self)`

Reply:
Thanks, @estebanmual, but doing that will cause an infinite method call loop, I think we should refactor that line as `item.author = self` and modify the add_* methods from the Item class for removing the prop assignation and leave it only to be in the Author, Genre, Label classes.